tools/fuzzer/symbol_table.cc

```diff
@@ -180,6 +205,17 @@ int calculate_freedom_index(lldb::SBValue value,
     }
   }
 
+  if (type.IsArrayType()) {
+    lldb::addr_t address =
+        static_cast<lldb::addr_t>(value.AddressOf().GetValueAsUnsigned());
+    if (is_valid_address(address, memory_regions)) {
+      // The first array element.
+      lldb::SBValue element_value = value.CreateValueFromAddress(
+          "value", address, type.GetArrayElementType());
```

Comment:
Maybe it's better to use `SBValue::GetChildAtIndex` for this? For arrays it returns the element at specified index.